Test/PerformanceCollector/FunctionalTests/Test40.cs

```diff
@@ -75,6 +76,15 @@ public void NonExistentCounter()
 
         [TestMethod]
         [Owner("alkaplan")]
+        [Description("Tests that non existent counters which use placeholders are not collected and wont affect other counters")]
+        [DeploymentItem(TestWebApplicaionSourcePath, TestWebApplicaionDestPath)]
+        public void NonExistentCounterWhichUsesPlaceHolder()
+        {
+            CommonTests.NonExistentCounterWhichUsesPlaceHolder(this.Listener);
+        }
+
+        [TestMethod]
+        [Owner("alkaplan")]
```

Comment:
Dont know the answer - i guess our team should be owning all the tests in this module.